src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -14,54 +15,23 @@
-    <_MauiImplicitPackageReference Include="Microsoft.Maui.Resizetizer" Version="$(MauiVersion)" PrivateAssets="all" />
-  </ItemGroup>
-  <ItemGroup Condition=" '$(UseMauiEssentials)' == 'true' ">
-    <_MauiImplicitPackageReference Include="Microsoft.Maui.Essentials" Version="$(MauiVersion)">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </_MauiImplicitPackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(UseMauiCore)' == 'true' ">
-    <_MauiImplicitPackageReference Include="Microsoft.Maui.Core" Version="$(MauiVersion)">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </_MauiImplicitPackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' ">
-    <_MauiImplicitPackageReference Include="Microsoft.Maui.Controls" Version="$(MauiVersion)">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </_MauiImplicitPackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' and '$(UseMauiCompat)' == 'true' ">
-    <_MauiImplicitPackageReference Include="Microsoft.Maui.Controls.Compatibility" Version="$(MauiVersion)">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </_MauiImplicitPackageReference>
-  </ItemGroup>
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' and '$(UsingMicrosoftNETSdkRazor)' == 'true' ">
-    <_MauiImplicitPackageReference Include="Microsoft.AspNetCore.Components.WebView.Maui" Version="$(MauiVersion)">
-      <PrivateAssets Condition=" '$(OutputType)' == 'Library' and '$(AndroidApplication)' != 'true' ">all</PrivateAssets>
-    </_MauiImplicitPackageReference>
-  </ItemGroup>
-  <!-- Always install the required dependecies, such as build tools for the SDK itself -->
-  <ItemGroup>
-    <_MauiRequiredImplicitPackageReference Remove="@(PackageReference)" />
-    <PackageReference Include="@(_MauiRequiredImplicitPackageReference)" />
-  </ItemGroup>
-  <!-- Install the nuget packages if we are supposed to -->
   <ItemGroup Condition=" '$(UseMauiNuGets)' != 'true' ">
-    <_MauiImplicitPackageReference Remove="@(PackageReference)" />
-    <PackageReference Include="@(_MauiImplicitPackageReference)" />
+    <_MauiAddedImplicitPackageReference Include="@(PackageReference->HasMetadata('IsMauiImplicitPackageReference'))" />
+    <_MauiAddedUserPackageReference Include="@(PackageReference->WithMetadataValue('IsMauiImplicitPackageReference',''))" />
+    <_MauiMissingImplicitPackageReference Include="@(_MauiAddedImplicitPackageReference)" Exclude="@(_MauiAddedUserPackageReference)" />
+    <PackageReference Remove="@(PackageReference)" />
+    <PackageReference Include="@(_MauiAddedUserPackageReference);@(_MauiMissingImplicitPackageReference)" />
   </ItemGroup>
```

Comment:
So we fix that here by removing the duplicates that _we_ added. If a developer references a package then we can detect that as it will not have the `IsMauiImplicitPackageReference` metadata. This now fixes the double Includes (from us and from the csproj)